<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Time</class>
 <widget class="QMainWindow" name="Time">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>219</width>
    <height>318</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Time</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../src/imgs/standar.png</normaloff>../src/imgs/standar.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{background:qlineargradient(spread:pad, x1:0.221909, y1:1, x2:0.601909, y2:0.119318, stop:0 rgba(102, 241, 73, 29), stop:1 rgba(255, 255, 255, 255))}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	background:transparent;}
#label_text,#label_result{
	font: 75 20pt &quot;Rockwell Condensed&quot;;
	background:none;
}#label_result{
	font: 16pt &quot;Rockwell Condensed&quot;;
}#comboBox1,#comboBox2{
	border:none;
	background:none;
}
#comboBox1:checked,#comboBox2:checked{
	background:none;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_text">
         <property name="text">
          <string>0</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox1">
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="currentIndex">
          <number>4</number>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
         <item>
          <property name="text">
           <string>Microseconds</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Milliseconds</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Seconds</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Minutes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hours</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Days</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Weeks</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Years</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_result">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox2">
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>40</height>
          </size>
         </property>
         <property name="palette">
          <palette>
           <active/>
           <inactive/>
           <disabled/>
          </palette>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="currentIndex">
          <number>3</number>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
         <item>
          <property name="text">
           <string>Microseconds</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Milliseconds</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Seconds</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Minutes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hours</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Days</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Weeks</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Years</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget{background:transparent;}
QPushButton{
	color:black;
	background:white rgba(255, 255, 255, 0.8);
	border:0;
	offline:none;
	margin:0px;
	font: 500 16pt &quot;Arial Black&quot;;
}
QPushButton:hover{
	background:rgba(0, 0, 0, 0.1);
}
#btn_clear,#btn_del_one{	font: 12pt &quot;Arial&quot;;padding:0px 2px;}
#btn_clear{color:red;}
#btn_del_one:hover{color:red}






</string>
      </property>
      <layout class="QGridLayout" name="_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <property name="spacing">
        <number>1</number>
       </property>
       <item row="2" column="1">
        <widget class="QPushButton" name="btn2">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="btn3">
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btn8">
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btn5">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="btn0">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btn1">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btn7">
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btn6">
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btn9">
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btn4">
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="btn_dot">
         <property name="text">
          <string>â€¢</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QWidget" name="widget_3" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget{background:rgba(255,255,255,0.8)}
QPushButton{background:transparent}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btn_del_one">
            <property name="text">
             <string>Â«</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_clear">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>CE</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>219</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionStandar"/>
    <addaction name="actionScientific"/>
    <addaction name="actionLenght"/>
    <addaction name="actionTime"/>
    <addaction name="separator"/>
    <addaction name="actionProperties"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionFQAs"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionStandar">
   <property name="text">
    <string>Standar</string>
   </property>
  </action>
  <action name="actionScientific">
   <property name="text">
    <string>Scientific</string>
   </property>
  </action>
  <action name="actionLenght">
   <property name="text">
    <string>Lenght</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionFQAs">
   <property name="text">
    <string>FQAs</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionTime">
   <property name="text">
    <string>Time</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
